tests/testthat/test-renderer1-knit-print.R

```diff
@@ -1,168 +1,197 @@
-
-test_that("clicking top legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(0, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-test_that("clicking bottom legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 0))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-plot1top$show_hide$clickElement()
-s.div <- plot1top$widget$findChildElement("class name", "selectize-input")
-s.div$clickElement()
-
-test_that("top widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(5, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(0, 10))
-  remDr$sendKeysToActiveElement(list("a", key="enter"))
-  expect_equal(get_circles(), list(5, 10))
-  remDr$sendKeysToActiveElement(list("b", key="enter"))
-  expect_equal(get_circles(), list(10, 10))
-})
-
-plot1bottom$show_hide$clickElement()
-s.div <-
-  plot1bottom$widget$findChildElement("class name", "selectize-input")
-s.div$clickElement()
-
-test_that("bottom widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(10, 5))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(10, 0))
-  remDr$sendKeysToActiveElement(list("a", key="enter"))
-  expect_equal(get_circles(), list(10, 5))
-  remDr$sendKeysToActiveElement(list("b", key="enter"))
-  expect_equal(get_circles(), list(10, 10))
-})
+acontext("knitting multiple animint plots in a single Rmd")
+print("In test")
+knitr::knit_meta() #clear knitr 'metadata'
+# Rmd.file <- "~/R/animint/inst/examples/test_knit_print.Rmd" ## Do we need this???
+Rmd.file <- system.file("inst", "examples", "test_knit_print.Rmd", 
+                         package = "animint2")
+Rmd.file <- file.path("..", "..", "inst","examples", "test_knit_print.Rmd")
+index.file <- file.path("animint-htmltest", "index.Rmd")
+
+if (!dir.exists(dirname(index.file))) {
+  # Create directory if it doesn't exist
+  dir.create(dirname(index.file), recursive = TRUE)
+}
+if (!file.exists(index.file)) {
+  # Create index.file if it doesn't exist
+  file.create(index.file)
+}
+
```

Comment:
please undo these additions which should not be necessary

Reply:
Thanks, these additions are reverted as well